cmd/utils/flags.go

```diff
@@ -2530,30 +2547,9 @@ func MakeChain(ctx *cli.Context, stack *node.Node, readonly bool) (*core.BlockCh
 		gspec   = MakeGenesis(ctx)
 		chainDb = MakeChainDatabase(ctx, stack, readonly)
 	)
-	cliqueConfig, err := core.LoadCliqueConfig(chainDb, gspec)
-	if err != nil {
-		Fatalf("%v", err)
-	}
-	ethashConfig := ethconfig.Defaults.Ethash
 
-	// ETC-specific configuration: ECIP1099 modifies the original Ethash algo, doubling the epoch size.
-	if gspec != nil && gspec.Config != nil {
-		ethashConfig.ECIP1099Block = gspec.GetEthashECIP1099Transition() // This will panic if the genesis config field is not nil.
-	}
+	engine := CreateConsensusEngine(ctx, stack, gspec, chainDb)
```

Comment:
(Oops, left a comment on this line but on the commit link and now that's hard to find and won't show up in the Converstation)... it went something like:

I like the idea of the refactor! Thanks! How about renaming `CreateConsensusEngine` to `MakeConsensusEngine` here to follow the `Make*` pattern for config/context-setup functions and to avoid conflating it with `ethconfig.CreateConsensusEngine` ?

Reply:
Yes that does stick to the continuety better and it defers to the Create* functions.